tests/ReviewsWidget.test.jsx

```diff
@@ -1,13 +1,39 @@
 import React from 'react';
 // import renderer from 'react-test-renderer';
 import { shallow, mount } from 'enzyme';
```

Comment:
why lowercase when the others are uppercase?  I'm sure I do this too but I'm wondering if there's a technical difference or...why we do it!  :D

Reply:
Same as mentioned before, following conventions. Don't think it'd be a mistake to use uppercase tho.